.taskcluster.yml

```diff
@@ -0,0 +1,36 @@
+version: 0
+tasks:
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events:
+          - pull_request.closed
+          - pull_request.opened
+          - pull_request.reopened
+          - pull_request.synchronize
+          - pull_request.assigned
+          - pull_request.unassigned
+          - pull_request.labeled
+          - pull_request.unlabeled
+          - pull_request.edited
+          - push
+          - release
+          
+    payload:
+      maxRunTime: 3600
+      image: node
```

Comment:
This is the docker image that the tests will run in, described at https://hub.docker.com/_/node/.  Since `package.json` says we use Node 8, we should test with that, too.  We can use the latest version of Node 8 with the image tag `node:8`.